autoload/ctrlsf/win.vim

```diff
@@ -119,6 +119,13 @@ func! s:InitMainWindow() abort
     setl textwidth=0
     setl nospell
     setl nofoldenable
+    if exists("*g:CtrlSFCustomizeWindowFunction")
+        if exists("b:ctrlsf_customized_window_function_called")
```

Comment:
this callback should be considered as a part of initialization, I think no need to add this variable besides of `b:ctrlsf_initialized`.